src/components/sbb-divider/sbb-divider.stories.js

```diff
@@ -0,0 +1,121 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+import { SbbColorBlackDefault, SbbColorWhiteDefault } from '@sbb-esta/lyne-design-tokens';
+
+const documentationPlatformContainerStyle = (context) => {
+  if (context.args.negative) {
+    return {};
+  }
+
+  return `background-color: ${SbbColorWhiteDefault};`;
+};
+
+const wrapperStyle = (context) => {
+  if (context.args.negative) {
+    return `background-color: ${SbbColorBlackDefault};`;
```

Comment:
```suggestion
    return `background-color: ${SbbColorCharcoalDefault};`;
```
not important, but normally in dark mode it is charcoal and not black.